x-pack/plugins/fleet/server/types/models/output.ts

```diff
@@ -9,16 +9,74 @@ import { schema } from '@kbn/config-schema';
 
 import { outputType } from '../../../common/constants';
 
+export function validateLogstashHost(val: string) {
```

Comment:
it would be nice to cover all branches with unit tests

Reply:
Just added some test